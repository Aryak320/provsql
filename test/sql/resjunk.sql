\set ECHO none
\pset format unaligned
SET search_path TO provsql_test, provsql;

CREATE TABLE resjunk AS
SELECT
    city,
    provenance ()
FROM (
    SELECT
        p1.city
    FROM
        personnel p1
        JOIN personnel p2 ON p1.city = p2.city
    GROUP BY
        p1.city,
        p2.name,
        p2.id) t;

SELECT
    remove_provenance ('resjunk');

ALTER TABLE resjunk
    DROP COLUMN provenance;

SELECT
    *
FROM
    resjunk
ORDER BY
    city;

DROP TABLE resjunk;
