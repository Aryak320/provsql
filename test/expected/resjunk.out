\set ECHO none
remove_provenance

(1 row)
city
Berlin
Berlin
New York
New York
Paris
Paris
Paris
(7 rows)
